mesonbuild/envconfig.py

```diff
@@ -57,6 +58,7 @@
     's390x',
     'sparc',
     'sparc64',
+    'xc16',
```

Comment:
This should not be here. Xc16 is a compiler toolchain, not a cpu family.

Reply:
updated this